app/cookies/RecurringContributionCookie.scala

```diff
@@ -7,7 +7,7 @@ import play.api.mvc.Cookie
 
 object RecurringContributionCookie {
 
-  val currentTime = Instant.ofEpochSecond(System.currentTimeMillis / 1000).toString
+  val currentTime = Instant.ofEpochSecond(System.currentTimeMillis / 1000).toEpochMilli.toString
```

Comment:
ha, it occurs to me now that this could just be `System.currentTimeMillis`...

Reply:
Monday

Reply:
https://trello.com/c/LIzM8Rza/702-clean-up-after-https-githubcom-guardian-support-frontend-pull-849